spec/models/post_spec.rb

```diff
@@ -60,6 +60,23 @@
         expect( post.points ).to eq(@up_votes - @down_votes)
       end
     end
+  end
+
+    describe "#create_vote" do
+      it "sets the post up_votes to 1" do
+        expect(post.up_votes).to eq(1)
+      end
+
+      it "calls #create_vote when a post is created" do
```

Comment:
Style issue, not crazy about putting this spec here because a call back describes the behaviour of a post more than the method. I would probably put this spec in a different describe or context block.